src/foam/dao/CachingDAO.js

```diff
@@ -135,11 +154,26 @@ foam.CLASS({
         delete this.private_['delegate'];
       }
       else if ( this.PurgeRecordCmd.isInstance(obj) ) {
+        // REVIEW: this.cache is a dao not object, need to call dao.remove(obj)?
         delete this.cache[obj.id];
       }
       else {
         this.SUPER(x, obj);
       }
+    },
+
+    function poll() {
+      var self = this;
+
+      self.delegate
+        .orderBy(self.pollingComparator).limit(1)
+        .select().then(function(data) {
+          if ( data.array.length === 1 ) {
+            self.src
+              .where(self.GT(self.src.of.ID, data.array[0].id))
```

Comment:
what is this?

Reply:
Refactored to polling property https://github.com/kgrgreer/foam3/pull/881/commits/16a898c91654fca6478bfc928148bc3c052f9d94.